aws/sagemaker/llm-instance-recommend/lmi_recommender/lmi_config_recommender.py

```diff
@@ -111,8 +126,13 @@ def category_checker(target, category):
     return found
 
 
-def rolling_batch_chooser(model_category: str, instance: dict, dtype: str):
+def strategy_chooser(model_category: str, instance: dict, dtype: str):
     instance_name = instance['name']
+    smoothquant_checker = dynamic_batch_deepspeed['smoothquant']
+    if dtype == 'int8':
+        if category_checker(model_category, smoothquant_checker['model_category']):
+            if category_checker(instance['name'], smoothquant_checker['instance']):
```

Comment:
nit: You can use `instance_name` rather than looking up `instance['name']` again